examples/demos/example23-immutable-data-grouping.js

```diff
@@ -1,13 +1,11 @@
+import faker from 'faker';
```

Comment:
Do we want to just delete this file?

Reply:
If they don't show something unique compared to other demos, we can remove them.

Reply:
IMO let's not edit js files without tsifying them. You can revert individual files like this:
```sh
git fetch
git checkout origin/pre-canary -- examples/demos/example23-immutable-data-grouping.js
```

Reply:
Those immutable / group files were changed before your addon removal PR, so the changes were kept here as well. I thought it's no harm to keep them as they were working. I will try to clean them then.